core/src/spclient.rs

```diff
@@ -567,7 +573,15 @@ impl SpClient {
 
     pub async fn get_metadata(&self, scope: &str, id: &SpotifyId) -> SpClientResult {
         let endpoint = format!("/metadata/4/{}/{}", scope, id.to_base16()?);
-        self.request(&Method::GET, &endpoint, None, None).await
+        // For unknown reasons, metadata requests must now be sent through spclient.wg.spotify.com.
+        // Otherwise, the API will respond with 500 Internal Server Error responses.
+        // Context: https://github.com/librespot-org/librespot/issues/1527
+        let options = RequestOptions {
```

Comment:
There is no need to always create an instance of it. Orient on `NO_METRICS_AND_SALT` and create a similar const.